pkg/kubelet/remote/fake/fake_runtime.go

```diff
@@ -53,6 +53,16 @@ func NewFakeRemoteRuntime() *RemoteRuntime {
 	return f
 }
 
+// SetStatus set fake remote runtime's network and runtime condition.
+func (f *RemoteRuntime) SetStatus() {
```

Comment:
Can we set it as part of Start() method?
Given that for now noone is actually overwriting it anywhere, this seems enough to me.

nit: you may also consider moving the changes to fake_runtime to a separate PR.

Reply:
> Can we set it as part of Start() method?
> Given that for now noone is actually overwriting it anywhere, this seems enough to me.

ok
> nit: you may also consider moving the changes to fake_runtime to a separate PR.

Do you mean a pre-pr of this one which modify fake_runtime.go and fake_runtime_service.go?

Reply:
Yes (those seem to be universal fixes needed independently from this PR).

Reply:
done